sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -804,6 +805,10 @@ def begin_analyze_batch_actions(  # type: ignore
                 key_phrase_extraction_tasks=[
                     t.to_generated() for t in
                     [a for a in actions if _determine_action_type(a) == AnalyzeBatchActionsType.EXTRACT_KEY_PHRASES]
+                ],
```

Comment:
can you update the typehint on line 734?

Reply:
Updated.  I also had to disable `line-too-long` for pylint on this one.